src/theory/sets/theory_sets.cpp

```diff
@@ -149,6 +149,16 @@ TrustNode TheorySets::ppRewrite(TNode n, std::vector<SkolemLemma>& lems)
       throw LogicException(ss.str());
     }
   }
+  if (n.getKind() == SET_MINUS && n[1].getKind() == SET_MINUS
+      && n[1][0] == n[0])
+  {
+    // note this cannot be a rewrite rule, since it impacts the cardinality
+    // graph.
+    // (setminus A (setminus A B)) = (intersection A B)
```

Comment:
```suggestion
    // (set.minus A (set.minus A B)) = (set.inter A B)
```

Reply:
Can you document more about this? for example these minus terms would be missing from the cardinality graph which may affect Venn regions reasoning.